lib/matplotlib/__init__.pyi

```diff
@@ -70,14 +70,22 @@ def get_cachedir() -> str: ...
 def get_data_path() -> str: ...
 def matplotlib_fname() -> str: ...
 
-class RcParams(dict[str, Any]):
+class RcParams(MutableMapping):
     validate: dict[str, Callable]
+    namespaces: tuple
+    single_key_set: set
     def __init__(self, *args, **kwargs) -> None: ...
+    def _split_key(self, key: str, sep: str = ".") -> tuple[list, int]: ...
+    def _set(self, key: str, val: Any) -> None: ...
+    def _get(self, key: str) -> Any: ...
     def __setitem__(self, key: str, val: Any) -> None: ...
     def __getitem__(self, key: str) -> Any: ...
+    def __delitem__(self, key: str) -> None: ...
```

Comment:
This is actually inherited from MutableMapping, but doesn't hurt to put it here more explicitly...

Reply:
I added these mainly because we overwrite these functions. So, just to be consistent with having stubs for the functions implemented..